sbin/overc-cctl

```diff
@@ -252,6 +248,20 @@ EOF
 		# Domain 0 wants to drive X onto a real console
 		echo "lxc.console = none" >>${pathtocontainer}/config
 	fi
+
+	if [ ! -z ${subuid} ]; then
+		# unprivileged containers
+		echo "lxc.mount.auto = cgroup:mixed proc:mixed sys:mixed" >>${pathtocontainer}/config
```

Comment:
"mixed" option will isolate the container user itself from the outside world. Separate hierarchy will be created to be used by this container only.